lib/TH/CMakeLists.txt

```diff
@@ -70,28 +70,6 @@ IF (CORTEXA9_FOUND)
   SET(CMAKE_C_FLAGS "-mcpu=cortex-a9 ${CMAKE_C_FLAGS}")
 ENDIF (CORTEXA9_FOUND)
 
-IF(UNIX)
-  INCLUDE(CheckFunctionExists)
```

Comment:
FindBlas needs this include. This caused build failure on my side. I'll fix it later.